pkg/cli/admin/policy/modify_roles.go

```diff
@@ -29,10 +29,10 @@ import (
 var (
 	addRoleToUserExample = templates.Examples(`
 		# Add the 'view' role to user1 for the current project
-		oc adm policy add-role-to-user view user1
+		oc policy add-role-to-user view user1
```

Comment:
@soltysh Unrelated to doc generation, but I'm just curious why `adm` is being removed from these examples. Aren't these examples for the `oc adm policy *` commands?

Reply:
@bergerhoffer they are used for both `oc policy` and `oc adm policy`, there are 2 possibilities how to fix this, either have a flag which presents the appropriate prefix, or just use `oc policy` which will be available for every user (admin and non-admin). I went with the latter, for simplicity :sweat_smile:

Reply:
Thanks @soltysh. Do the commands do the exact same things whether an admin uses "oc adm policy" or "oc policy"?